eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -496,7 +496,7 @@ data TxBodyRaw era = TxBodyRaw
     _vldt :: !ValidityInterval,
     _update :: !(StrictMaybe (Update era)),
     _reqSignerHashes :: !(Set (KeyHash 'Witness (Crypto era))),
-    _mint :: !(MaryValue (Crypto era)),
+    _mint :: !(MultiAsset (Crypto era)),
     -- The spec makes it clear that the mint field is a
     -- Cardano.Ledger.Mary.Value.MaryValue, not a Value.
     -- Operations on the TxBody in the BabbageEra depend upon this.
```

Comment:
Same as the comment in the Alonzo era.  Clear up the comments.
-- The spec makes it clear that the mint field is a
-- Cardano.Ledger.Mary.Value.MaryValue, not a Cardano.Ledger.Core.Value. (the type family)
-- Operations on the TxBody in the AlonzoEra depend upon this. We now store only the MultiAsset part of a Mary.Value.